libraries/path_helpers.rb

```diff
@@ -0,0 +1,24 @@
+# Encoding: utf-8
+
+module PathHelpers
```

Comment:
General comment... should we be name-spacing our classes and modules in a module named for the cookbook so that generic names like "PathHelpers" aren't accidentally reused or worse accidentally monkey patched elsewhere?